x-pack/test/functional/config_security_basic.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/* eslint-disable import/no-default-export */
+
+import { resolve } from 'path';
+
+import { services } from './services';
+import { pageObjects } from './page_objects';
+
+// the default export of config files must be a config provider
+// that returns an object with the projects config values
+export default async function({ readConfigFile }) {
+  const kibanaCommonConfig = await readConfigFile(
+    require.resolve('../../../test/common/config.js')
+  );
+  const kibanaFunctionalConfig = await readConfigFile(
+    require.resolve('../../../test/functional/config.js')
+  );
+
+  return {
+    // list paths to the files that contain your plugins tests
+    testFiles: [resolve(__dirname, './apps/security/basic_license')],
+
+    services,
+    pageObjects,
+
+    servers: kibanaFunctionalConfig.get('servers'),
+
+    esTestCluster: {
+      license: 'basic',
+      from: 'snapshot',
+      serverArgs: [],
+    },
+
+    kbnTestServer: {
+      ...kibanaCommonConfig.get('kbnTestServer'),
+      serverArgs: [
+        ...kibanaCommonConfig.get('kbnTestServer.serverArgs'),
+        '--status.allowAnonymous=true',
+        '--server.uuid=5b2de169-2785-441b-ae8c-186a1936b17d',
+        '--xpack.maps.showMapsInspectorAdapter=true',
+        '--xpack.maps.preserveDrawingBuffer=true',
+        '--xpack.reporting.queue.pollInterval=3000', // make it explicitly the default
+        '--xpack.reporting.csv.maxSizeBytes=2850', // small-ish limit for cutting off a 1999 byte report
```

Comment:
Supernit: are all of these config args necessary? Seems like cruft.

I know this was copied from `xpack/test/functional/config.js`, but since the only test file that relies on this is `basic_license`, maybe the extraneous args could be removed. I can just imagine this being a bit confusing in the future.